guides/source/association_basics.md

```diff
@@ -1658,7 +1658,7 @@ Controls what happens to the associated objects when their owner is destroyed:
 
 * `:destroy` causes all the associated objects to also be destroyed
 * `:delete_all` causes all the associated objects to be deleted directly from the database (so callbacks will not execute)
-* `:nullify` causes the foreign keys to be set to `NULL`. Callbacks are not executed.
+* `:nullify` causes the foreign key to be set to `NULL`. On polymorphic associations both the foreign key and foreign type column will be set to `NULL`. Callbacks are not executed. 
```

Comment:
Ditto.

Reply:
@kamipo what does Ditto mean?

Reply:
> @kamipo what does Ditto mean?

/column/columns/, and ✂️ a trailing space.